src/components/addForm/addForm.css

```diff
@@ -0,0 +1,56 @@
+form {
+  display: flex;
+  flex-direction: column;
+}
+
+fieldset {
+  display: flex;
+  justify-content: space-between;
+  padding: 1em;
+  margin: 2em;
+  border: none;
+  font-weight: var(--fw-primary-light);
+}
+
+#today,
+#tomorrow,
+#choose-date {
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+}
```

Comment:
Good idea, although it seems to have a somewhat patchy support 
https://caniuse.com/?search=appearance